awl/mslider.cpp

```diff
@@ -23,6 +23,8 @@
 
 namespace Awl {
 
+#define METER_LEFT_EDGE 18
```

Comment:
note, I don't know your coding rules for constant values, so I can change to const int or however else.
I put at top of .cpp file because I want this to be visible outside of both functions that need to follow this constant.